src/ioctls/mod.rs

```diff
@@ -15,6 +15,10 @@ use std::result;
 use kvm_bindings::kvm_run;
 #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
 use kvm_bindings::{kvm_cpuid2, kvm_cpuid_entry2};
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+use vmm_sys_util::fam::{FamStruct, FamStructWrapper};
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
I think it is better to move the definition of MAX_KVM_CPUID_ENTRIES here instead of importing it from `src/lib.rs`.

Reply:
Makes sense. Done.